ui/ui-components/components/record/list.tsx

```diff
@@ -416,18 +373,26 @@ export default function RecordsList(props) {
 }
 
 RecordsList.hydrate = async (
-  ctx,
+  ctx: NextPageContext,
   _searchKey?: string,
   _searchValue?: string
 ) => {
   const { execApi } = UseApi(ctx);
-  const { id, limit, offset, state, searchKey, searchValue, caseSensitive } =
-    ctx.query;
+  const {
+    id,
+    modelId,
+    limit,
+    offset,
+    state,
+    searchKey,
+    searchValue,
+    caseSensitive,
+  } = ctx.query;
 
   let groupId: string;
   if (id) {
     if (ctx.pathname.match("/group/")) {
-      groupId = id;
+      groupId = id as string;
```

Comment:
```suggestion
      groupId = String(id);
```
Safer way to do this if you'd like